src/eureka/S6_planet_spectra/s6_spectra.py

```diff
@@ -383,7 +395,7 @@ def plot_spectra(eventlabel, ecf_path=None, s5_meta=None, input_meta=None):
 
             log.closelog()
 
-    return meta
+    return meta, lc
```

Comment:
Please add this extra returned variable to the Returns section of the function's docstring